backend/editor/models.py

```diff
@@ -23,51 +23,51 @@ class Footer(Marginal):
 
 # Models for FastAPI
 
-class CommonParameters:
-    branch = "branch"
-    taxonomy_name = "taxonomy_name"
 
-class Entry:
-    default = []
-    title: str
-    description: str
+class CommonParameters(BaseModel):

```

Comment:
I have added description, changed the syntax of importing models, resolved the long impending error this PR had.